viziphant/statistics.py

```diff
@@ -41,3 +41,72 @@ def plot_isi(intervals, label, binsize=2*pq.ms, cutoff=250*pq.ms):
 
     return fig, ax
 
+
+def plot_patterns_statistics(patterns, winlen, binsize, n_neurons):
+    """
+    This function creates a histogram plot to visualise patterns statistics
+    output of a SPADE analysis.
+
+    Parameters
+    ----------
+    patterns : list
+        The output of elephant.spade
+    winlen : int
```

Comment:
the `winlen` parameter is not clear here:
* it should be either 1 or what?
* can we rename it to something like `with_delays` or...
* can we extract information on whether "only synchronous patterns" or "patterns with delays" were explored from the output of spade?

Reply:
- The winlen param can be any integer, from 1 on. If winlen == 1, then all patterns are composed of synchronous spikes. Otherwise, they might contain delays. 
- I was unsure whether it is better to add a flag like `with_delays` as you said, or to not let the user think too much, and obtain it just from the window length of the spade analysis. 
- Unfortunately, from the output of spade it is not really possible to extract the information on the synchrony or not, since it might be by chance that only synchronous patterns are detected, even if the analysis was intended to include also patterns with delays. In that case the histogram would be misleading because the fourth panel would not show up...

Reply:
yeah, this is a topic of provenance objects - wouldn't it be nice to have one object as the output of spade that carries over all necessary info about the parameters and arguments used...
I'll think of a better name, if any, once I get it through.